core/domain/user_services.py

```diff
@@ -1729,3 +1798,203 @@ def is_topic_manager(user_id):
+        language_code)
+    _update_user_community_rights(user_community_rights)
+
+
+def allow_user_to_review_voiceover_in_language(user_id, language_code):
+    """Allows the user with the given user id to review voiceover applications
+    in the given language_code.
+
+    Args:
+        user_id: str. The unique ID of the user.
+        language_code: str. The code of the language. Callers should ensure that
+            the user does not have rights to review voiceovers in the given
+            language code.
+    """
+    user_community_rights = get_user_community_rights(user_id)
+    language_set = set(
+        user_community_rights.can_review_voiceover_for_language_codes)
+    language_set.add(language_code)
+    sorted_language_list = sorted(list(language_set))
+    user_community_rights.can_review_voiceover_for_language_codes = (
+        sorted_language_list)
+    _save_user_community_rights(user_community_rights)
+
+
+def remove_voiceover_review_rights_in_language(user_id, language_code):
+    """Removes the user's review rights to voiceover applications in the given
+    language_code.
+
+    Args:
+        user_id: str. The unique ID of the user.
+        language_code: str. The code of the language. Callers should ensure that
+            the user have rights to review voiceovers in the given
+            language code.
+    """
+    user_community_rights = get_user_community_rights(user_id)
+    user_community_rights.can_review_voiceover_for_language_codes.remove(
+        language_code)
+    _update_user_community_rights(user_community_rights)
+
+
+def allow_user_to_review_question(user_id):
+    """Allows the user with the given user id to review question suggestions.
+
+    Args:
+        user_id: str. The unique ID of the user. Callers should ensure that
+            the given user does not have rights to review questions.
+    """
+    user_community_rights = get_user_community_rights(user_id)
+    user_community_rights.can_review_questions = True
+    _save_user_community_rights(user_community_rights)
+
+
+def remove_question_review_rights(user_id):
+    """Removes the user's review rights to question suggestions.
+
+    Args:
+        user_id: str. The unique ID of the user. Callers should ensure that
+            the given user have rights to review questions.
+    """
+    user_community_rights = get_user_community_rights(user_id)
+    user_community_rights.can_review_questions = False
+    _update_user_community_rights(user_community_rights)
+
+
+def remove_community_reviewer(user_id):
+    """Deletes the UserCommunityRightsModel corresponding to the given user_id.
+
+    Args:
+        user_id: str. The unique ID of the user.
+    """
+    user_community_rights_model = (
+        user_models.UserCommunityRightsModel.get_by_id(user_id))
+    if user_community_rights_model is not None:
+        user_community_rights_model.delete()
+
+
+def get_community_reviewer_usernames(review_type, language_code=None):
```

Comment:
If you are using "review category" (based on the naming of the constants) then let's use that throughout instead of review type.